### PR TITLE
[PDI-13408] - Metastore performance degrades with each object added - Set AutomaticXmlMetaStoreCache as default cache.+ Add Guava lib rev - 17

### PR DESCRIPTION
**Pull request by AndreyBurikhin**
@mdamour1976  please review.
I have just one question. Should I create new request for IP Audit (or something like this) if add new library to metastore project? It is the same version of guava as in kettle. But let me know if have to get additional approval from IP Audit Team before you'll merge this pull request.

### CODE REVIEW

File: ivy.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
   <dependencies defaultconf="default->default">
 
     <dependency org="commons-codec"       name="commons-codec"       rev="1.5"              transitive="false"/>
+    <dependency org="com.google.guava"    name="guava"               rev="17.0"             transitive="false"/>
```
Comment (mattyb149) [inline ivy.xml]: This should probably use a variable for revision, specifically the variable and value used in Kettle.

**Comment by mdamour1976** [comment]
@AndreyBurikhin that version already is in use in the in many places by pentaho distributions.  If it was new, then yes.  

If you are going to take care of the @mattyb149 comment, go ahead and use variables for all other revisions in that file too.

**Comment by AndreyBurikhin** [comment]
@mdamour1976 , @mattyb149 please review.

**Comment by mdamour1976** [comment]
Looks good, can you squash again?

**Comment by AndreyBurikhin** [comment]
@mdamour1976 done

**Pull request merged**